### PR TITLE
i18n(ko-KR): update `i18n.mdx`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### Description

- update `i18n.mdx`
- #1620 
- #1798

### CODE REVIEW

File: docs/src/content/docs/ko/guides/i18n.mdx [hunk 1]
```diff
@@ -143,6 +143,35 @@ Starlight는 귀하가 모든 언어로 같은 페이지를 생성할 것을 기
 
 특정 언어에 대한 번역이 아직 제공되지 않는 경우 Starlight는 독자에게 해당 페이지의 콘텐츠를 기본 언어(`defaultLocale`을 통해 설정된 언어)로 표시합니다. 예를 들어, About 페이지의 프랑스어 버전을 아직 만들지 않았고 기본 언어가 영어인 경우 `/fr/about`에 방문한 독자는 이 페이지가 아직 번역되지 않았다는 알림과 함께 `/en/about`에 있는 영어 콘텐츠를 보게 됩니다. 이를 통해 기본 언어로 콘텐츠를 추가한 다음 점진적으로 번역할 수 있습니다.
 
+## 사이트 제목 번역
+
+기본적으로 Starlight는 모든 언어에 대해 동일한 사이트 제목을 사용합니다.
+
+각 로케일의 제목을 사용자 정의해야 하는 경우 Starlight 옵션의 [`title`](/ko/reference/configuration/#title-필수)에 객체를 전달할 수 있습니다.
```
Comment (HiDeoo) [inline docs/src/content/docs/ko/guides/i18n.mdx:150]:
```suggestion
기본적으로 Starlight는 모든 언어에 대해 동일한 사이트 제목을 사용합니다.
각 로케일의 제목을 사용자 정의해야 하는 경우 Starlight 옵션의 [`title`](/ko/reference/configuration/#title-필수)에 객체를 전달할 수 있습니다.
```

Small nit to match the English version.

File: docs/src/content/docs/ko/guides/i18n.mdx [hunk 2]
```diff
@@ -143,6 +143,35 @@ Starlight는 귀하가 모든 언어로 같은 페이지를 생성할 것을 기
...
+	integrations: [
+		starlight({
+-			title: 'My Docs',
++			title: {
++				en: 'My Docs',
++				'zh-CN': '我的文档',
++			},
+			defaultLocale: 'en',
+			locales: {
+				en: { label: 'English' },
```
Comment (HiDeoo) [inline docs/src/content/docs/ko/guides/i18n.mdx:167]: Would it make sense like the example right above to change the default locale from `en` to `ko` for consistency and translate "My Docs" with "나의 문서"? Let me know what you think.

**Review by HiDeoo — COMMENTED** [review]
Thanks for the translation :thumbsup:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view). I left one question regarding consistency with the example above this new one, but other than that, everything looks good to me.

File: docs/src/content/docs/ko/guides/i18n.mdx (hunk 2 above)
Comment (jsparkdev) [reply docs/src/content/docs/ko/guides/i18n.mdx]: I forgot to edit it. thank you!

File: docs/src/content/docs/ko/guides/i18n.mdx (hunk 1 above)
Comment (jsparkdev) [reply docs/src/content/docs/ko/guides/i18n.mdx:150]: Thank you!

**Review by HiDeoo — APPROVED** [review]
Thanks for the fast update :raised_hands:

**Pull request merged**